src/qt/README.md

```diff
@@ -0,0 +1,90 @@
+This directory contains the BitcoinQT graphical user interface (GUI). It uses the cross platform framework [QT](https://www1.qt.io/developers/).
+
+The current precise version for QT 5 is specified in [qt.mk](/depends/packages/qt.mk). QT 4 is also supported (see [#8263](https://github.com/bitcoin/bitcoin/issues/8263)).
+
+## Compile and run
+
+See build instructions ([OSX](/doc/build-osx.md), [Windows](/doc/build-windows.md), [Unix](/doc/build-unix.md), etc). To compile just BitcoinQT:
+
+```sh
+make src/qt/bitcoin-qt
+```
+
+It can be compiled without wallet support: 
+
+```sh
+./configure --disable-wallet
+```
+
+To run:
+
+```sh
+./src/qt/bitcoin-qt
+```
+
+To create a DMG for OSX:
+
+```sh
+make deploy
+```
+
+## Files and directories
+
+### forms
+
+Contains [Designer UI](http://doc.qt.io/qt-5.9/designer-using-a-ui-file.html) files.
+
+### locale
+
+Contains translations. They are periodically updated. The process is described [here](/doc/translation_process.md). 
+
+## res
+
+Resources such as the icon.
+
+## test
+
+Tests.
```

Comment:
Not sure if listing folders whose description can be inferred from the name is useful.

Reply:
It's a placeholder. I should explain how to run them, roughly what these tests (don't) cover, and perhaps point to some work in progress on UI tests.